nimbus.go

```diff
@@ -3,15 +3,33 @@
 package main
 
 import (
-	statusnim "github.com/status-im/status-nim"
-	"runtime"
+	"crypto/ecdsa"
+
+	"github.com/jroimartin/gocui"
+	nimbusbridge "github.com/status-im/status-protocol-go/bridge/nimbus"
+	whispertypes "github.com/status-im/status-protocol-go/transport/whisper/types"
 )
 
 func init() {
-	runtime.LockOSThread()
+	nimbusbridge.Init()
```

Comment:
`init` might not the be most fortunate choice because you don't have any control when it's called.

Reply:
Still, do you anticipate any issues? I think we're fine as long as it runs before `Main`.

Reply:
`nimbusbridge.Init()` calls `runtime.LockOSThread()`. `init()` is called in the main thread. If nimbus uses always mainthread also then I guess all is fine.

Reply:
Yeah, I've implemented a `RoutineQueue` class which ensures that all Nimbus calls are made from the main thread.